site/content/en/docs/Installation/Install Agones/confirm.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Confirming Agones Installation"
+linkTitle: "Confirm Installation"
+weight: 40
+description: >
+  Confirming Agones is installed and started successfully.
```

Comment:
nit: Verify that Agones is installed and has started successfully.

I find having the same word on top of itself on two lines looks strange, so I'd suggest using a synonym for the description so that it doesn't exactly match the title.

Reply:
Alternately, change the title to verify...